src/pages/mobile/Intro.vue

```diff
@@ -31,6 +31,12 @@
       >
         {{ $t('intro.onboarding') }}
       </AeButton>
+      <div class="terms-and-conditions">
+        {{ $t('terms-and-conditions.note') }}
+        <RouterLink :to="{ name: 'terms-and-conditions' }">
+          {{ $t('terms-and-conditions.link') }}
+        </RouterLink>
+      </div>
```

Comment:
I propose to make it as in this example: https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage